### ISSUE TITLE
ConfigMissingCredentialsError after update to 0.166.1-1.1

### ISSUE THREAD

**Issue by maltek** [openSUSE/osc#679]
Hi,

I've been using the keyring to store credentials before the update, and after updating I get the following exception:

```
ibs config --change-password
Traceback (most recent call last):
  File "/usr/bin/osc", line 39, in <module>
    osccli = commandline.Osc()
  File "/usr/lib/python3.7/site-packages/osc/commandline.py", line 91, in __init__
    self._load_plugins()
  File "/usr/lib/python3.7/site-packages/osc/commandline.py", line 9325, in _load_plugins
    mod = imp.load_source(modname, os.path.join(plugin_dir, extfile))
  File "/usr/lib64/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/mkraus/.osc-plugins/osc-mq.py", line 166, in <module>
    osc.conf.get_config()
  File "/usr/lib/python3.7/site-packages/osc/conf.py", line 901, in get_config
    raise oscerr.ConfigMissingCredentialsError('No password found in section %s' % url, conffile, url)
osc.oscerr.ConfigMissingCredentialsError
```

**Comment by maltek** [comment]
I just realized the stacktrace points into the plugins. Without the plugins, there is no exception - I get asked for credentials. I'm not sure if that's intended.

**Comment by marcus-h** [comment]
On 2019-11-14 07:45:32 -0800, maltek wrote:
> I just realized the stacktrace points into the plugins. Without the plugins, there is no exception - I get asked for credentials. I'm not sure if that's intended.
>
The plugin should not call get_config (this is only needed if in case of
an external python module that is not supposed to be an osc plugin).
0.166.1 has a bug wrt. the keyring handling. This is fixed in 0.166.2.

**Comment by maltek** [comment]
OK, thanks. I guess I can close this then.
